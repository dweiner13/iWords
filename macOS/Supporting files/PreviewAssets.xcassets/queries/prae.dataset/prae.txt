prae                 ADV    POS                         
prae  ADV  
before, in front of; forward [prae sequor => go on before];
prae                 PREP   ABL                         
prae  PREP  ABL  
before, in front; in view of, because of;

