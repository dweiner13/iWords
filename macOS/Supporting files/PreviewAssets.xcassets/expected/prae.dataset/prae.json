[{"words":[{"meaning":"before, in front of; forward [prae sequor => go on before];","expansion":{"adv":{"_0":"prae","_1":[]}}}],"truncated":false,"possibilities":[{"adverb":{"_0":{"degree":"POS","text":"prae"}}}]},{"words":[{"meaning":"before, in front; in view of, because of;","expansion":{"prep":{"_0":"prae","_2":[],"_1":"ABL"}}}],"truncated":false,"possibilities":[{"preposition":{"_0":{"text":"prae","case":"ABL"}}}]}]